microsoft/PowerToys
# Added localization to default sizes

## Summary of the Pull Request

**What is this about:**
If the Image Resizer tab on settings was opened default sizes were not localized. Because names of sizes are editable we decided to store them localized to the settings file. Another option was to make default sizes as readonly but that solution would be messy and hard to accomplish.

**What is include in the PR:** 
- Add `GetSettings` method to `SettingsUtils.cs`
- Rename `GetSettings` method to `GetSettingsDefault`
- Localize default sizes for settings 

**How does someone test / validate:** 
- Delete settings.json from ImageResizer AppData folder
- Open PT settings on the Image Resizer tab
- Check if the settings.json contains localized strings instead of hardcoded ones

One can do a check for regression by deleting the settings.json file and opening the image resizer and settings in different orders.

## Quality Checklist

- [X] **Linked issue:** #7432 

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
